### PR TITLE
Bump Lighthouse version and add flag for verbose logging

### PR DESCRIPTION
**Pull request by chloewakefieldd**
## Background

Getting intermittent failure on Jenkins BlueOcean when running a11y-tests-web:
```TypeError: Cannot destructure property 'audits' of 'results' as it is undefined```

## Resolution

- Bump Google Lighthouse dependency to ^2.9.4 from ^2.7.0
- Change logging output level from silent to verbose
- Add additional logging in case of future errors

### CODE REVIEW

File: test/lib/lighthouse.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function getMinifiedMatcher(code) {
 }
 
 const EXPECTED_LIGHTHOUSE_FLAGS = {
-  logLevel: 'silent',
+  logLevel: 'verbose',
```
Comment (andymsuk) [inline test/lib/lighthouse.js:24]: What extra logging do we get from this? Can you give an example output please (removing anything confidential)?

Comment (chloewakefieldd) [reply test/lib/lighthouse.js:24]: There's ... a lot - probably easiest if I link you to the builds:

Full verbose lighthouse logging:
https://ci.iplayer.tools.bbc.co.uk/job/test-a11y-master/34/consoleFull

Previous silent lighthouse logging:
https://ci.iplayer.tools.bbc.co.uk/job/test-a11y/10/console

Comment (chloewakefieldd) [reply test/lib/lighthouse.js:24]: Extract from the verbose lighthouse logging:
```Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request started. 12 requests in progress (56 started and 44 finished).
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.requestWillBeSentExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.requestWillBeSentExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.requestWillBeSentExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.requestWillBeSentExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.requestWillBeSentExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.requestWillBeSentExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request finished. 11 requests in progress (56 started and 45 finished).
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request finished. 10 requests in progress (56 started and 46 finished).
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request finished. 9 requests in progress (56 started and 47 finished).
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request finished. 8 requests in progress (56 started and 48 finished).
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request finished. 7 requests in progress (56 started and 49 finished).
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:22 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:22 GMT NetworkRecorder:verbose Request finished. 6 requests in progress (56 started and 50 finished).
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:23 GMT NetworkRecorder:verbose Request finished. 5 requests in progress (56 started and 51 finished).
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:23 GMT NetworkRecorder:verbose Request finished. 4 requests in progress (56 started and 52 finished).
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceivedExtraInfo 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:23 GMT NetworkRecorder:verbose Request finished. 3 requests in progress (56 started and 53 finished).
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.loadingFinished 
Wed, 12 Aug 2020 14:01:23 GMT NetworkRecorder:verbose Request finished. 2 requests in progress (56 started and 54 finished).
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.dataReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.responseReceived 
Wed, 12 Aug 2020 14:01:23 GMT <= event:verbose Network.dataReeived 
￼Wed, 12 Aug 2020 14:01:23 GMT method => browser:verbose Runtime.evaluate 
Wed, 12 Aug 2020 14:01:23 GMT method <= browser OK:verbose Runtime.evaluate 
Wed, 12 Aug 2020 14:01:23 GMT Driver:verbose CPU has been idle for 5326.135000000249 ms
Wed, 12 Aug 2020 14:01:23 GMT Driver:verbose loadEventFired and network considered idle
Wed, 12 Aug 2020 14:01:23 GMT statusEnd Loading page & waiting for onload
Wed, 12 Aug 2020 14:01:23 GMT status Retrieving devtoolsLog and network records
Wed, 12 Aug 2020 14:01:23 GMT NetworkRecorder:verbose Request started. 1 requests in progress (1 started and 0 finished).
Wed, 12 Aug 2020 14:01:23 GMT NetworkRecorder:verbose Request finished. 0 requests in progress (1 started and 1 finished).
Wed, 12 Aug 2020 14:01:23 GMT GatherRunner:error failed document request (net::ERR_NETWORK_CHANGED) https://www.test.bbc.co.uk/iplayer
Wed, 12 Aug 2020 14:01:23 GMT status Disconnecting from browser...
[31m✘[0m [31mCould not run lighthouse[0m
[31m✘[0m [31mTypeError: Cannot destructure property 'audits' of 'results' as it is undefined.[0m
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @bbc/a11y-tests-web@6.2.0 start:lighthouse:junit-headless: `A11Y_HEADLESS=true node runLighthouse.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @bbc/a11y-tests-web@6.2.0 start:lighthouse:junit-headless script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /var/lib/jenkins/.npm/_logs/2020-08-12T14_01_23_758Z-debug.log
```

Previous silent lighthouse logging:

```Mon, 10 Aug 2020 15:19:58 GMT status Auditing: Custom controls have associated labels
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: Custom controls have ARIA roles
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: User focus is not accidentally trapped in a region
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: Interactive controls are keyboard focusable
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: Interactive elements indicate their purpose and state
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: The page has a logical tab order
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: The user's focus is directed to new content added to the page
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: Offscreen content is hidden from assistive technology
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: HTML5 landmark elements are used to improve navigation
Mon, 10 Aug 2020 15:19:58 GMT status Auditing: Visual order on the page follows DOM order
Mon, 10 Aug 2020 15:19:58 GMT status Generating results...
[31m✘[0m [31mTypeError: Cannot read property 'total' of undefined[0m
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @bbc/a11y-tests-web@6.2.0 start:lighthouse:junit-headless: `A11Y_HEADLESS=true node runLighthouse.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @bbc/a11y-tests-web@6.2.0 start:lighthouse:junit-headless script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /var/lib/jenkins/.npm/_logs/2020-08-10T15_19_58_558Z-debug.log```

Comment (andymsuk) [reply test/lib/lighthouse.js:24]: That's quite a lot of logging! Maybe we want to make it an option?

Comment (chloewakefieldd) [reply test/lib/lighthouse.js]: Changed to be a flag usage ```A11Y_LOGGING_LEVEL='verbose'``` or leave blank for silent

**Pull request closed without merging**